index.js

```diff
@@ -0,0 +1,37 @@
+const database = require('./lib/database');
+
+database.connect('./db');
+
+const models = require('./models/models');
+
+const person = {
+  firstName: { type: 'string', required: true },
+  lastName: { type: 'string', required: true },
+  married: { type: 'boolean' },
+  kids: { type: 'number' }
+};
+
+models.create(person)
+  .then(res => {
+    models.findById(res.id)
+      .then(res => console.log(res));
+  });
+
+models.find()
```

Comment:
this can also be in a .then after findById so that you can make sure it happens after the model is created. When it is outside, it could potentially get called before .create and .findById are finished.